cmd/describe/machinepool/options.go

```diff
@@ -13,25 +13,31 @@ type DescribeMachinepoolUserOptions struct {
 type DescribeMachinepoolOptions struct {
 	reporter *reporter.Object
 
-	args DescribeMachinepoolUserOptions
+	args *DescribeMachinepoolUserOptions
 }
 
-func NewDescribeMachinepoolUserOptions() DescribeMachinepoolUserOptions {
-	return DescribeMachinepoolUserOptions{machinepool: ""}
+func NewDescribeMachinepoolUserOptions() *DescribeMachinepoolUserOptions {
+	return &DescribeMachinepoolUserOptions{machinepool: ""}
 }
 
 func NewDescribeMachinepoolOptions() *DescribeMachinepoolOptions {
 	return &DescribeMachinepoolOptions{
 		reporter: reporter.CreateReporter(),
-		args:     DescribeMachinepoolUserOptions{},
+		args:     &DescribeMachinepoolUserOptions{},
 	}
 }
 
 func (m *DescribeMachinepoolOptions) Machinepool() string {
 	return m.args.machinepool
 }
 
-func (m *DescribeMachinepoolOptions) Bind(args DescribeMachinepoolUserOptions) error {
+func (m *DescribeMachinepoolOptions) Bind(args *DescribeMachinepoolUserOptions, argv []string) error {
```

Comment:
@hunterkepley Can we make it a bit easier to read this method ;-)

We have 3 different ways to refer to the same thing I think 🤔 

Does this work:

```
func (m *DescribeMachinepoolOptions) Bind(args *DescribeMachinepoolUserOptions, argv []string) error {
 m.args = args
 if m.args.machinepool == "" {
  if len(argv) > 0 {
   m.args.machinepool = argv[0]
  }
 }

 if m.args.machinepool == "" {
   return fmt.Errorf("you need to specify a machine pool name")
 }

return nil
}
```